src/services/ubi/community.ts

```diff
@@ -1582,4 +1548,152 @@ export default class CommunityService {
             metrics: metrics !== null ? [metrics] : undefined,
         };
     }
+
+    private static _generateInclude(fields: any): Includeable[] {
+        const extendedInclude: Includeable[] = [];
+        if (fields.suspect) {
+            extendedInclude.push({
+                model: this.ubiCommunitySuspect,
+                attributes:
+                    fields.suspect.length > 0 ? fields.suspect : undefined,
+                as: 'suspect',
+                required: false,
+                duplicating: false,
+                where: {
+                    createdAt: {
+                        [Op.eq]: literal(
+                            '(select max("createdAt") from ubi_community_suspect ucs where ucs."communityId"="Community".id and date("createdAt") > (current_date - INTERVAL \'1 day\'))'
+                        ),
+                    },
+                },
+            });
+        }
+
+        if (fields.cover || fields.thumbnails) {
+            extendedInclude.push({
+                attributes: fields.cover
+                    ? fields.cover.length > 0
+                        ? fields.cover
+                        : undefined
+                    : ['id'],
+                model: this.appMediaContent,
+                as: 'cover',
+                duplicating: false,
+                include: fields.thumbnails
+                    ? [
+                          {
+                              model: this.appMediaThumbnail,
+                              as: 'thumbnails',
+                              separate: true,
+                          },
+                      ]
+                    : [],
+            });
+        }
```

Comment:
```suggestion
        if (fields.cover) {
            extendedInclude.push({
                attributes: ['url'],
                model: this.appMediaContent,
                as: 'cover',
                duplicating: false,
                include: [
                      {
                          attributes: ['url', 'width', 'height', 'pixelRatio'],
                          model: this.appMediaThumbnail,
                          as: 'thumbnails',
                          separate: true,
                      },
                  ],
            });
        }
```
What I mean is this. We always need all these fields. And besides these fields, there are only ids, which are not needed on the UI. So, there are not really optional parameters here.
Does it make sense?